documentation/sphinx/source/administration.rst

```diff
@@ -633,59 +633,24 @@ You can now remove old client library versions from your clients. This is only t
 Version-specific notes on upgrading
 ===================================
 
-Upgrading from 5.1.x
+Upgrading from 5.2.x
 --------------------
 
-Upgrades from 5.0.x will keep all your old data and configuration settings. 5.1 has a new backup format so backups will need to be restarted after upgrading.
+Upgrades from 5.2.x will keep all of your old data and configuration settings. 
```

Comment:
It looks you added an `of` here but not in the other two remaining locations that mention keeping all of your old data. I feel like they should be consistent, all either all "all of" or all "all".